src/main/kotlin/gogo/gogostage/domain/community/root/application/CommunityServiceImpl.kt

```diff
@@ -0,0 +1,22 @@
+package gogo.gogostage.domain.community.root.application
+
+import gogo.gogostage.domain.community.board.application.BoardProcessor
+import gogo.gogostage.domain.community.root.application.dto.WriteCommunityBoardDto
+import gogo.gogostage.global.util.UserContextUtil
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class CommunityServiceImpl(
+    private val communityReader: CommunityReader,
+    private val boardProcessor: BoardProcessor,
+    private val userUtil: UserContextUtil
+): CommunityService {
+
+    @Transactional
+    override fun writeCommunityBoard(gameId: Long, writeCommunityBoardDto: WriteCommunityBoardDto) {
+        val student = userUtil.getCurrentStudent()
+        val community = communityReader.readCommunityByGameId(gameId)
+        boardProcessor.saveCommunityBoard(community, student.studentId, writeCommunityBoardDto)
+    }
```

Comment:
```suggestion
    @Transactional
    override fun writeCommunityBoard(gameId: Long, writeCommunityBoardDto: WriteCommunityBoardDto) {
        val student = userUtil.getCurrentStudent()
        val community = communityReader.readCommunityByGameId(gameId)
        boardProcessor.saveCommunityBoard(community, student.studentId, writeCommunityBoardDto)
        
        // 욕설 필터링 요청 처리 필요
    }
```

추후에 게시글 작성, 댓글 작성시 비동기 이벤트 통신으로 AI 서비스에 욕설 필터링 처리가 필요하여 주석으로 메모해두는게 좋아보입니다!

Reply:
[af7fb6f](https://github.com/team-gogo/GOGO-Stage/pull/10/commits/af7fb6f462cc7b19ff32e5d12b9059bb2aa6a9be)
주석으로 표시하였습니다